outputs.tf

```diff
@@ -1,3 +1,18 @@
+output "cloudwatch_log_group" {
+  description = "All outputs from `aws_cloudwatch_log_group.default`"
+  value       = aws_cloudwatch_log_group.default
+}
+
+output "cloudwatch_log_group_arn" {
+  description = "Cloudwatch log group ARN"
+  value       = try(aws_cloudwatch_log_group.default[0].arn, "")
+}
+
+output "cloudwatch_log_group_name" {
+  description = "Cloudwatch log group name"
+  value       = try(aws_cloudwatch_log_group.default[0].name, "")
+}
```

Comment:
when we have `cloudwatch_log_group`, why even keep `cloudwatch_log_group_arn` and `cloudwatch_log_group_name`? Is it possibly even a bad practice to use all outputs of a certain resource because the outputs might change?

Reply:
I guess it was just copy&paste from the old module